packages/core/src/lavadome.mjs

```diff
@@ -9,10 +9,24 @@ import {
     appendChild,
     replaceChildren,
     textContentSet,
+    navigation,
+    url, destination, includes,
+    preventDefault, stopPropagation,
 } from './native.mjs';
 import {distraction, unselectable} from './element.mjs';
 import {getShadow} from './shadow.mjs';
 
+// text-fragments links can be abused to leak shadow internals - block in-app redirection to them
+navigation.addEventListener('navigate', event => {
```

Comment:
(could) maybe add an option to opt-out, starting with `unsafe` ?

Reply:
I want to create software that's as less dependent on configuration as possible, meaning I'd support configuration only when it's truly necessary